packages/components/nodes/chains/GraphCypherQAChain/GraphCypherQAChain.ts

```diff
@@ -0,0 +1,273 @@
+        this.version = 2.0
+        this.type = 'GraphCypherQAChain'
+        this.icon = 'graphqa.svg'
+        this.category = 'Chains'
+        this.description = 'Advanced chain for question-answering against a Neo4j graph by generating Cypher statements'
+        this.baseClasses = [this.type, ...getBaseClasses(GraphCypherQAChain)]
+        this.sessionId = fields?.sessionId
+        this.inputs = [
+            {
+                label: 'Language Model',
+                name: 'model',
+                type: 'BaseLanguageModel',
+                optional: true,
+                description: 'Model for generating Cypher queries and answers.'
+            },
+            {
+                label: 'Cypher Generation Model',
+                name: 'cypherModel',
+                optional: true,
+                type: 'BaseLanguageModel',
+                description: 'Model for generating Cypher queries. If not provided, the main model will be used.'
+            },
+            {
+                label: 'QA Model',
+                name: 'qaModel',
+                optional: true,
+                type: 'BaseLanguageModel',
+                description: 'Model for generating answers. If not provided, the main model will be used.'
+            },
+            {
+                label: 'Neo4j Graph',
+                name: 'graph',
+                type: 'Neo4j'
+            },
+            {
+                label: 'Memory',
+                name: 'memory',
+                type: 'BaseMemory',
+                optional: true,
+                description: 'Memory to store chat history for context'
+            },
+            {
+                label: 'Cypher Generation Prompt',
+                name: 'cypherPrompt',
+                type: 'BasePromptTemplate',
+                description: 'Prompt template for generating Cypher queries. Must include {schema} and {question} variables'
+            },
+            {
+                label: 'QA Prompt',
+                name: 'qaPrompt',
+                optional: true,
+                type: 'BasePromptTemplate',
+                description: 'Prompt template for generating answers. Must include {context} and {question} variables'
+            },
+            {
+                label: 'Input Moderation',
+                description: 'Detect text that could generate harmful output and prevent it from being sent to the language model',
+                name: 'inputModeration',
+                type: 'Moderation',
+                optional: true,
+                list: true
+            },
+            {
+                label: 'Return Direct',
+                name: 'returnDirect',
+                type: 'boolean',
+                default: false,
+                optional: true,
+                description: 'If true, return the raw query results instead of using the QA chain'
+            },
+            {
+                label: 'Return Intermediate Steps',
```

Comment:
Turning on this doesn't have any difference though, still seeing the same streaming output, prob can remove this

Reply:
sure, intermediate steps generates a new array type field intermediateSteps in which the query and the data returned from neo4j come for the specific case of flowise it would not be necessary, I have removed it